src/react-apps/package.json

```diff
@@ -4,44 +4,11 @@
   "scripts": {
     "install-deps": "lerna bootstrap --hoist",
     "no-ci-hoist": "lerna bootstrap --hoist --no-ci",
-    "test": "jest",
-    "install-all": "lerna exec -- npm i"
+    "test": "lerna exec -- npm test",
+    "install-all": "lerna exec -- npm i",
+    "clean-all": "lerna exec -- rm package-lock.json"
```

Comment:
"lerna clean" is deleting all "node_modules" folders. Should this "script-name" to something that is so different?
How about "clean-lock"?

Reply:
Renamed to `clean-lock`.